applications/luci-app-upnp/htdocs/luci-static/resources/view/upnp/upnp.js

```diff
@@ -153,18 +153,18 @@ return view.extend({
 		o.datatype    = 'uinteger'
 		o.placeholder = 30
 
-		o = s.taboption('advanced', form.Value, 'clean_ruleset_threshold', _('Clean rules threshold'))
+		o = s.taboption('advanced', form.Value, 'clean_ruleset_threshold', _('Clean rules threshold of number of rules cleaned once'))
```

Comment:
Keep the name as is but add a description instead:

```
o = s.taboption('advanced', form.Value, 'clean_ruleset_threshold', _('Clean rules threshold'), _('Clean not used port forwardings once their number reached the threshold'))
```

Reply:
Yeah, subtext - keep setting title to three words or fewer.

Reply:
o = s.taboption('advanced', form.Value, 'clean_ruleset_threshold', _('Ruleset cleanse interval'), _('Clean unused port forwards every X minutes. 0 is disabled.'))

Reply:
```js
o = s.taboption('advanced', form.Value, 'clean_ruleset_threshold', _('Ruleset cleanse threshold'), _('Start to remove unused rules when the amount of rules exceeds this value.'))
```

Reply:
Expired is better.

Reply:
Fix the PR checks please.

Reply:
expired means expired.
@stokito is right about the threshold.